QTMSettings.cs

```diff
@@ -1680,4 +1736,22 @@ public enum TimestampType
         [XmlEnum("CameraTime")]
         CameraTime,
     }
+    /// <summary>Degree of freedom</summary>
+    public enum DegreeOfFreedomType
+    {
+        [XmlEnum("Unknown DegreeOfFreedom")]
+        Unknown = -1,
+        [XmlEnum("RotationX")]
+        RotationX = 0,
+        [XmlEnum("RotationY")]
+        RotationY,
```

Comment:
Might aswell give all enums their respective int value. To make it easier to read.

Reply:
Done.